.vscode/settings.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
add this to .gitignore and uncommit. maybe just toss a comment `#vscode` if it doesn't exist and place .vscode folder underneath it